packages/design-tokens/src/figma/darkTheme.json

```diff
@@ -19,7 +19,7 @@
       "description": "For raised neutral surface above background/default."
     },
     "default-hover": {
-      "value": "#1A1B1C",
+      "value": "#1a1b1c",
```

Comment:
Same color. Just changing to lowercase. We are pretty inconsistent with this across the board so we can resolve in a enablement sprint